selene_sdk/update_seqweaver.py

```diff
@@ -0,0 +1,152 @@
+"""
+This module provides the `UpdateSeqweaver` class, which wraps the master bed file
+containing all of the features' binding sites parsed from CLIP-seq.
+It supports new dataset construction and training for Seqweaver.
+
+"""
+import h5py
+import gzip
+import numpy as np
+import sys
+
+from sequences.genome import Genome
+from targets.genomic_features import GenomicFeatures
+from samplers.dataloader import H5DataLoader
+from train_model import TrainModel
+from utils.config import load_path
+from utils.config_utils import parse_configs_and_run
+
+class UpdateSeqweaver():
+    """
+    Stores a dataset specifying sequence regions and features.
+    Accepts a tabix-indexed `*.bed` file with the following columns,
+    in order:
+        [chrom, start, end, strand, feature]
```

Comment:
i do think this BED file format would be more desirable when including strand information (vs having `chrom, start, end, feature, strand`) but it'd be a more substantial change to GenomicFeatures, so maybe we can think about it as a future TODO...

Reply:
I definitely agree that the previous order is more intuitive, this was somewhat of a quick fix. I thought about doing this without changing the column order but as you say, it will require substantial changes to GenomicFeatures. I would definitely like to go back to this soon :-)